feature/experimental/bgp/otg_tests/link_bandwidth_test/link_bandwidth_test.go

```diff
@@ -0,0 +1,832 @@
+	policy.SetImportPolicy([]string{policyName})
+	gnmi.Replace(t, dut, path.Config(), policy)
+	// apply V6 policy
+	path = gnmi.OC().NetworkInstance(dni).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).Bgp().Neighbor(atePort1.IPv6).AfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST).ApplyPolicy()
+	policy = root.GetOrCreateNetworkInstance(dni).GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).GetOrCreateBgp().GetOrCreateNeighbor(atePort1.IPv6).GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST).GetOrCreateApplyPolicy()
+	policy.SetImportPolicy([]string{policyName})
+	gnmi.Replace(t, dut, path.Config(), policy)
+}
+func validatPolicyDut(t *testing.T, dut *ondatra.DUTDevice, policyName string) {
+	dni := deviations.DefaultNetworkInstance(dut)
+	path := gnmi.OC().NetworkInstance(dni).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).Bgp().Neighbor(atePort1.IPv4).AfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).ApplyPolicy()
+	policy := gnmi.Get[*oc.NetworkInstance_Protocol_Bgp_Neighbor_AfiSafi_ApplyPolicy](t, dut, path.State())
+	importPolicies := policy.GetImportPolicy()
+	if len(importPolicies) != 1 {
+		t.Fatalf("ImportPolicy = %v, want %v", importPolicies, []string{policyName})
+	}
+}
+func validateRouteCommunityV4(t *testing.T, td testData, ec extCommunity, localPerf bool) {
+	// To verify received route communities on ATE ports
+	log, ok := gnmi.WatchAll(t,
+		td.ate.OTG(),
+		gnmi.OTG().BgpPeer(td.otgP2.Name()+".BGP4.peer").UnicastIpv4PrefixAny().State(),
+		time.Minute,
+		func(v *ygnmi.Value[*otgtelemetry.BgpPeer_UnicastIpv4Prefix]) bool {
+			_, present := v.Val()
+			return present
+		}).Await(t)
+	t.Logf("log:%v", log)
+	if ok {
+		bgpPrefixes := gnmi.GetAll(t, td.ate.OTG(), gnmi.OTG().BgpPeer(td.otgP2.Name()+".BGP4.peer").UnicastIpv4PrefixAny().State())
+		t.Logf("bgp prefix:%v", bgpPrefixes)
+		for _, bgpPrefix := range bgpPrefixes {
+			if bgpPrefix.Address != nil && bgpPrefix.GetAddress() == v41Route {
+				t.Logf("Prefix recevied on OTG is correct, got prefix:%v Address %s, want prefix %v", bgpPrefix, *bgpPrefix.Address, v41Route)
+				if ec.prefixSet1Comm == "none" {
+					if len(bgpPrefix.Community) == 0 {
+						t.Logf("community is empty")
+					} else {
+						t.Fatalf("community is not empty it should be none")
+					}
+				} else if ec.prefixSet1Comm == "100:100" {
+					for _, gotCommunity := range bgpPrefix.Community {
+						t.Logf("community AS:%d val: %d", *gotCommunity.CustomAsNumber, *gotCommunity.CustomAsValue)
+						if *gotCommunity.CustomAsNumber != 100 && *gotCommunity.CustomAsValue != 100 {
+							t.Fatalf("community is not 100:100")
+						}
+					}
+				} else {
+					// TODO Verification as OTG not supported for Extended community
+					t.Logf("TODO")
```

Comment:
Please also add a deviation which queries for the following paths from the DUT to try and do verification.   (I ask for a deviation, because OTG is really what we want, but until then, a deviation will work for some vendors and is better than nothing).  

/network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv4-unicast/loc-rib/routes/route/state/ext-community-index
	
/network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/loc-rib/routes/route/state/ext-community-index

(I was able to find support for [Arista EOS 4.31.2F](https://www.arista.com/en/support/toi/path-support?pn=EOS-4.31.2F) which I mention because it's relatively easy to find this on their website)